src/selectedCategory/selectedDifficulty.ts

```diff
@@ -0,0 +1,99 @@
+import he from 'he'
+import $, { error } from 'jquery'
+import Swal from 'sweetalert2'
+import { quiz } from '../'
+import { checkAnswer } from '../checkAnswer'
+import { renderCategory } from '../renderQuiz'
+import { Question } from '../types'
+
+export const selectedDifficulty = () => {
+    $('.difficulty').click(event => {
+        const quizDifficulty = $('.quiz__difficulty')
+        const quizText = $('.quiz--text')
+        const target = event.target
+        const categoryID = $(target).attr('categoryid')
+        const difficultName = $(target).attr('name')
+
+        quizDifficulty.slideUp(1000)
+        quizText.slideUp(200)
+
+        setTimeout(() => $('.quiz--spinner').show(), 1000)
+
+        quizDifficulty.remove()
+        quizText.remove()
+        $('.quiz').addClass('questions')
+
+        const APIQUESTIONS = `https://opentdb.com/api.php?amount=10&category=${categoryID}&difficulty=${difficultName}&type=multiple`
+
+        setTimeout(() => {
+            fetch(APIQUESTIONS)
+                .then(res => {
+                    if (res.ok) {
+                        return res.json()
+                    }
+
+                    throw error
+                })
+                .then((question: Question) => {
+                    const { response_code, results } = question
+
+                    if (response_code === 0) {
```

Comment:
there is no response code = 0

Reply:
this is api code to help the programmer

Response Codes
The API appends a "Response Code" to each API Call to help tell developers what the API is doing.

Code 0: Success Returned results successfully.
Code 1: No Results Could not return results. The API doesn't have enough questions for your query. (Ex. Asking for 50 Questions in a Category that only has 20.)
Code 2: Invalid Parameter Contains an invalid parameter. Arguements passed in aren't valid. (Ex. Amount = Five)
Code 3: Token Not Found Session Token does not exist.
Code 4: Token Empty Session Token has returned all possible questions for the specified query. Resetting the Token is necessary.